### ISSUE TITLE
The acc's value is always 50.000000 when training.

### ISSUE THREAD

**Issue by ChengYeung1222** [WeiTang114/MVCNN-TensorFlow#25]
I use two classes(airplane and bathtub) of the modelnet40v1 dataset to train the MVCNN model.I set up 10 shapes per class in the train_lists.txt and 2 shapes per class for validation.Does anyone have the same problem?Is this a random guessing problem?

**Comment by WeiTang114** [comment]
Could you try with more training data?

**Comment by monapr** [comment]
I am having the same problem: I am currently training on 145 shapes and use 45 shapes for validation. While loss decreases, accuracy is always exactly 51.219512 and doesn't change even after 80 epochs. Have you found a solution?

**Comment by hassaanmahmood** [comment]
I also get this problem but with different dataset. loss is decreasing but accuracy is staying exactly at 50. after training I test the model with test, validation and training datasets, I got 50, 50, 46 respectively.
Anyone know what can be the problem?
Did anyone check with other implementations?.
thanks.
